FIDO2_authenticate_credential_rsp.md

```diff
@@ -0,0 +1,29 @@
+## FIDO2 Authenticate Credential Response
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Ditto